ios/nskeyedarchiver/unarchiver.go

```diff
@@ -11,7 +11,13 @@ import (
 // Primitives will be extracted just like regular Plist primitives (string, float64, int64, []uint8 etc.).
 // NSArray, NSMutableArray, NSSet and NSMutableSet will transformed into []interface{}
 // NSDictionary and NSMutableDictionary will be transformed into map[string] interface{}. I might add non string keys later.
-func Unarchive(xml []byte) ([]interface{}, error) {
+func Unarchive(xml []byte) (result []interface{}, err error) {
+	defer func() {
+		if r := recover(); r != nil {
+			err = fmt.Errorf("Unarchive: %s", r)
```

Comment:
you can make the error a bit more helpful with adding the stack to the error (https://pkg.go.dev/runtime/debug#Stack)

Reply:
Thank you, added.